dev-tex/classico/classico-20140828.ebuild

```diff
@@ -0,0 +1,44 @@
+# Copyright 2018 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit latex-package
+
+DESCRIPTION="LaTeX, pdfLaTeX, XeLaTeX and LuaLaTeX support for four URW Classico fonts"
```

Comment:
Is there really a purpose in listing all possible TeX variants here? We kinda get that it works with everything.